<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="widget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>1</y>
      <width>771</width>
      <height>611</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget::item { min-width: 50px; }</string>
    </property>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>45</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>35</number>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn_salvar_banco">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>550</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
background-color: rgb(48, 141, 255);
font-size: 20px;
}
QPushButton:hover{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
background-color:rgb(112, 169, 255);
font-size: 25px;
}</string>
    </property>
    <property name="text">
     <string>SALVAR BANCO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalvarCNPJS">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>490</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
background-color: rgb(48, 141, 255);
font-size: 20px;
}
QPushButton:hover{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
background-color:rgb(112, 169, 255);
font-size: 25px;
}</string>
    </property>
    <property name="text">
     <string>SALVAR SOZINHO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPesqAPI">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>70</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
background-color: rgb(48, 141, 255);
font-size: 20px;
}
QPushButton:hover{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
background-color:rgb(112, 169, 255);
font-size: 25px;
}</string>
    </property>
    <property name="text">
     <string>PESQUISAR(API)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelP">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>160</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelP_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>190</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelP_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>130</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ATUALIZAÇÃO A CADA 60 SEGUNDOS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPARAR">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
background-color: rgb(48, 141, 255);
font-size: 20px;
}
QPushButton:hover{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
background-color:rgb(112, 169, 255);
font-size: 25px;
}</string>
    </property>
    <property name="text">
     <string>PARAR</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
